src/Reflex/Process.hs

```diff
@@ -127,16 +143,21 @@ createProcess
   => CreateProcess
   -> ProcessConfig t ByteString
   -> m (Process t ByteString ByteString)
-createProcess = createRedirectedProcess input output output
-  where
+createProcess p processConfig = do
```

Comment:
Let's document as an `NB:` that this uses an *unbounded* channel to prevent the networking from blocking.